### PR TITLE
Add tasksh formula

### PR DESCRIPTION
**Pull request by gusennan**
Revived PR #37514

### CODE REVIEW

File: Library/Formula/tasksh.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/tasksh", "--version"
+    rcfile = open testpath/".taskrc", "w"
+    rcfile.write "data.location=#{testpath}/.task\n"
+    rcfile.close
```
Comment (xu-cheng) [inline Library/Formula/tasksh.rb]: Let's replace these three lines to `(testpath/".taskrc").write "data.location=#{testpath}/.task\n"`

**Comment by xu-cheng** [comment]
Please squash the commits into one with the message "tasksh 1.0.0 (new formula)". Thanks.

**Comment by gusennan** [comment]
@xu-cheng done.

**Comment by tdsmith** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**